src/fsharp/NameResolution.fs

```diff
@@ -3128,7 +3288,7 @@ let rec ResolveTypeLongIdentPrim sink (ncenv: NameResolver) occurence first full
 
 
 /// Resolve a long identifier representing a type and report it
-let ResolveTypeLongIdent sink (ncenv: NameResolver) occurence fullyQualified nenv ad (lid: Ident list) staticResInfo genOk =
+let ResolveTypeLongIdentAux sink (ncenv: NameResolver) occurence fullyQualified nenv ad (lid: Ident list) staticResInfo genOk =
```

Comment:
Consider marking as `private` to indicate more strongly that this is a helper?

Reply:
Marking this `private` won't do anything. everything below this function will still be able to access it.

Reply:
Yes, but not the point. This is more of a stylistic thing. `private` more strongly indicates that something is a helper.